AMW_angular/io/src/app/apps/app-server-add/app-server-add.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="modal-header">
+  <h5 class="modal-title" id="editReleaseTitle">Create application server</h5>
+  <button type="button" class="btn btn-light close" aria-label="Close" (click)="cancel()">
+    <span aria-hidden="true">&times;</span>
+  </button>
+</div>
```

Comment:
Same as in `app-add.component.html` (and probably others too) - this is a candidate for a separete component.